File: 프로그래머스/lv1/12919. 서울에서 김서방 찾기/서울에서 김서방 찾기.java
class Solution {
    public String solution(String[] seoul) {
        String answer = "";
        
        for(int i = 0; i < seoul.length; i++) {
			if(seoul[i].equals("Kim")) {
				answer += "김서방은 " + i + "에 있다";
			} // seoul의 i번째 원소가 Kim과 같으면 answer에 i를 입력한다
		}
        return answer;
    }
}

File: 프로그래머스/1/12919. 서울에서 김서방 찾기/서울에서 김서방 찾기.js
function solution(seoul) {
    let answer = "";

    for (let i = 0; i < seoul.length; i++) {
        if (seoul[i] === "Kim") {
            answer += "김서방은 " + i + "에 있다";
        } // seoul의 i번째 원소가 Kim과 같으면 answer에 i를 입력한다
    }
    return answer;
}
